media/js/firefox/whatsnew/whatsnew-134beta.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I added a `firefox_whatsnew_events` bundle in https://github.com/mozilla/bedrock/pull/15512 that can be re-used between pages rather than repeating this code every WNP